app/views/ReportingFrequencyView.scala.html

```diff
@@ -135,114 +168,152 @@
+        @messages("reporting.frequency.latency.details.section.text.para3.text.2")
     }
 }
 
-@latentBusinessDropdownContent = {
-    @p(id = Some("separately-choose-to-opt-out")) {
-        @messages("reporting.frequency.latency.details.section.text.para1")
+@differentObligations = {
+    @h2(msg = "reporting.frequency.different.obligations.heading", optId = Some("reporting-frequency-different-obligations-heading"))
+    @p(id = Some("reporting-frequency-different-obligations-p1")){
+        @messages("reporting.frequency.different.obligations.p1")
     }
-    <ul class="govuk-list govuk-list--bullet">
-        <li id="latency-section-1-bullet-1">@messages("reporting.frequency.latency.details.section.text.para1.bullet1")</li>
-        <li id="latency-section-1-bullet-2">@messages("reporting.frequency.latency.details.section.text.para1.bullet2")</li>
-    </ul>
-    @p(id = Some("options-available")) {
-        @messages("reporting.frequency.latency.details.section.text.para2")
-    }
-    <ul class="govuk-list govuk-list--bullet">
-        <li id="latency-section-2-bullet-1">@messages("reporting.frequency.latency.details.section.text.para2.bullet1")</li>
-        <li id="latency-section-2-bullet-2">@messages("reporting.frequency.latency.details.section.text.para2.bullet2")</li>
-        <li id="latency-section-2-bullet-3">@messages("reporting.frequency.latency.details.section.text.para2.bullet3")</li>
-    </ul>
+    <ul class="govuk-list govuk-list--bullet" id="reporting-frequency-different-obligations-ul">
 
-    @h3(
-        msg = messages("reporting.frequency.latency.details.section.text.para3.heading"),
-        optId = Some("change-reporting-obligations-heading")
-    )
+        <li id="reporting-frequency-different-obligations-li1">
+            @messages("reporting.frequency.different.obligations.li1")
+        </li>
 
-    @p(id = Some("your-businesses")) {
-        @messages("reporting.frequency.latency.details.section.text.para3.text.1")
-        @linkComponent(
-                id = Some("your-businesses-link"),
-                url = if(viewModel.isAgent) controllers.manageBusinesses.routes.ManageYourBusinessesController.showAgent().url else controllers.manageBusinesses.routes.ManageYourBusinessesController.show().url,
-                message = messages("reporting.frequency.latency.details.section.text.para3.link.text")
+        <li id="reporting-frequency-different-obligations-li2">
+            @messages("reporting.frequency.different.obligations.li2")
+        </li>
+
+    </ul>
+    @p(id = Some("reporting-frequency-different-obligations-p2")){
+        @messages("reporting.frequency.different.obligations.p2")
+    }
+    @p(id = Some("reporting-frequency-different-obligations-updates-and-deadlines")){
+        @messages("reporting.frequency.different.obligations.p3")
+        @linkComponent(nextUpdatesLink,
+            message = s"${messages("reporting.frequency.different.obligations.link.text")} ",
```

Comment:
I think we can remove the string interpolation here:

`message = s"${messages("reporting.frequency.different.obligations.link.text")} ",`

message = messages("reporting.frequency.mandatoryReporting.link.text")